src/Classes/OneClickItem.php

```diff
@@ -163,15 +107,23 @@ public function toArray()
     {
         $oneClickItem = array(
             'reference' => $this->reference,
+            'title' => $this->title,
+            'amount' => $this->amount->toArray(),
```

Comment:
Add check field is OystPrice ?

Reply:
The check is already made in constructor
![image](https://user-images.githubusercontent.com/861701/37029347-67a5f33a-2137-11e8-9f61-835f4b208b1c.png)

Reply:
Good